string trigger_name
string trigger_params # A JSON object with key-value pairs.
string action_name
string action_params
